### PR TITLE
My first pull request!

### PR DESCRIPTION
**Pull request by sjdev2212**
My first pull request 
In this repository I:
.created a branch named featured-1.
.created an index.html file and an h1 with the text "Hello Microverse"
. created a file named styles.css and gave style to the h1
. created a test.md and a gitignore file. added the test.md to the gitignore file along with node_modules.
. installed Lighthouse, Webhint and Stylelint and ran the tests.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+.test.md
+node_modules/
```
Comment (DanteAlonsoHT) [inline .gitignore]: - [ ] Instead of `node_modules/` you should type `node_modules`.

File: README.md [hunk 2]
```diff
@@ -1 +1,7 @@
-# Anibal-Amoroso
\ No newline at end of file
+# Anibal-Amoroso
+Copyright 2021, [Anibal Amoroso]
+Permission is hereby granted, free of charge, to any person obtaining a copy of this [webpage/website] and associated documentation files, to deal in the [webpage/website] without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the [APP TYPE], and to permit persons to whom the [webpage/website] is furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all copies or substantial portions of the [webpage/website].
+
+THE [webpage/website] IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE [webpage/website] OR THE USE OR OTHER DEALINGS IN THE [webpage/website].
```
Comment (DanteAlonsoHT) [inline README.md]: - [ ] You should use the [Template provided by Microverse](https://github.com/microverseinc/readme-template), remember you can remove unnecessary sections such as the "Run tests" and "Deployment" section 😉

**Review by DanteAlonsoHT — CHANGES_REQUESTED** [review]
<h1> Status: Required Changes 🚫 </h1>

Hi @sj1978, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
- [x] Improve the description in your Pull Request, explain what you did, what is the main scope, which files do you change or added, what you set up in your project in this pull request, and so on.
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Make a better description in your Pull Request.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by DanteAlonsoHT — CHANGES_REQUESTED** [review]
<h1>Status: Changes Required 🚫 ♻️  </h1>

Hi @sj1978, 

You didn't do the previous changes required 😢 Please implement these changes required before request a new code review. If you are stuck, please check [This Example](https://github.com/DanteAlonsoHT/Leaderboard_API) to let you know what we expect from students in their README files. 

Also, check this [Another Example about .gitignore file](https://github.com/DanteAlonsoHT/Leaderboard_API/blob/development/.gitignore) if you want, just add `test.md` as you currently have in your `.gitignore` file 😃 

 ## Required Changes ♻️
_Check the comments above the review._ 👆 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by Leon-Mbegera — APPROVED** [review]
Hi @sj1978 , 

**Approved** 🎉

Welldone for addressing the changes requested by the previous reviewer. 💪 

Your project is complete! Congratulations! 
 There is nothing else to say other than... it's time to merge it :shipit: 
 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**